src/components/menu/menu.css

```
@@ -0,0 +1,15 @@
+.menu-container {
+  display: flex;
+  justify-content: space-evenly;
+  align-items: center;
+}
+
+.menu-items {
+  display: flex;
+  align-items: center;
+  margin-top: 5px;
```

faz mt diferença, sera?
```suggestion
  margin-top: 4px;
```